### PR TITLE
Add tests for UserCheckboxes component

### PR DESCRIPTION
**Pull request by brokoli777**
Fixes: #151

Added tests to make sure the checkboxes are working as expected.

image

Please let me know if there are any changes to be made.

### CODE REVIEW

File: __tests__/components/ChatBotBody/UserCheckboxes.test.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+		(useSettingsContext as jest.Mock).mockReturnValue(mockSettingsContext);
+		(useStylesContext as jest.Mock).mockReturnValue(mockStylesContext);
+		(usePathsContext as jest.Mock).mockReturnValue(mockPathsContext);
+	});
+
+	afterEach(() => {
+		jest.clearAllMocks();
+	});
+
+	test("renders checkboxes correctly", () => {
```
Comment (tjtanjin) [inline __tests__/components/ChatBotBody/UserCheckboxes.test.tsx:66]: Let's standardize to use `it` instead of `test`.

**Review by tjtanjin — CHANGES_REQUESTED** [review]
Thanks for the PR! One small nitpick!

File: __tests__/components/ChatBotBody/UserCheckboxes.test.tsx (hunk 1 above)
Comment (brokoli777) [reply __tests__/components/ChatBotBody/UserCheckboxes.test.tsx]: Hi! I updated it to use "it"

**Review by tjtanjin — APPROVED** [review]
LGTM!

**Pull request merged**